docs/getting-started/ea-req.asciidoc

```diff
@@ -0,0 +1,37 @@
+[[ea-requirements]]
+= Entity Analytics prerequisites
+
+[discrete]
+== Privileges
+
+To enable the risk scoring engine, you need the following privileges:
+
+Cluster privileges:
+
+* `manage_index_templates`
+* `manage_transform`
+
+Index privileges:
+
+`all` privilege for `risk-score.risk-score-*`
+
+{kib} privileges:
+
+* **All** for the **Saved Objects Management** feature under **Management**
+* **Read** for the **Security** feature 
+
+[discrete]
+== {es} resource guidelines
+
+Follow these guidelines to ensure clusters have adequate memory to handle data volume:
+
+* With 2GB of Java Virtual Machine (JVM) heap memory, the risk scoring task can safely process around 44 million documents (30 days of risk data with an ingest rate of 1000 documents per minute).
```

Comment:
@SourinPaul similar question to what I asked above: is it ok to highlight some of these underlying components (risk score task), or should we stick with the more general/all-encompassing term of "risk score feature"?

@natasha-moore-elastic I wasn't sure if the information in the parentheses is an example of what 44 million docs looks like IRL, so feel free to reject my suggestion if it's wrong. Also, wow, being able to performantly process 44 million docs is bonkers -- in a good way ofc!

```suggestion
* With 2GB of Java Virtual Machine (JVM) heap memory, the risk scoring task can safely process around 44 million documents, or 30 days of risk data with an ingest rate of 1000 documents per minute.
```

Reply:
@nastasha-solomon let's replace the reference of the task with a more generic; maybe `risk scoring engine` (engine for consistency). Thoughts?

44M 🔥